src/ag-mock.js

```diff
@@ -0,0 +1,97 @@
+self.transcend = Object.assign({
```

Comment:
let's keep this in a test-utils folder to make it obvious what this should be used for